# Removing the div does not cause errors because nextcloud checks if it exists before initializing vue
diff --git a/core/routes.php b/core/routes.php
index 820db44bf90..3e8e1d763bd 100644
--- a/core/routes.php
+++ b/core/routes.php
@@ -84,8 +84,6 @@ $application->registerRoutes($this, [
 		['name' => 'Reference#preview', 'url' => '/core/references/preview/{referenceId}', 'verb' => 'GET'],
 		['name' => 'Css#getCss', 'url' => '/css/{appName}/{fileName}', 'verb' => 'GET'],
 		['name' => 'Js#getJs', 'url' => '/js/{appName}/{fileName}', 'verb' => 'GET'],
-		['name' => 'contactsMenu#index', 'url' => '/contactsmenu/contacts', 'verb' => 'POST'],
-		['name' => 'contactsMenu#findOne', 'url' => '/contactsmenu/findOne', 'verb' => 'POST'],
 		['name' => 'WalledGarden#get', 'url' => '/204', 'verb' => 'GET'],
 		['name' => 'Search#search', 'url' => '/core/search', 'verb' => 'GET'],
 		['name' => 'Wipe#checkWipe', 'url' => '/core/wipe/check', 'verb' => 'POST'],
diff --git a/core/templates/layout.user.php b/core/templates/layout.user.php
index 11f7656cc91..6ca5575eca3 100644
--- a/core/templates/layout.user.php
+++ b/core/templates/layout.user.php
@@ -76,7 +76,6 @@ $getUserAvatar = static function (int $size) use ($_): string {
 			<div class="header-right">
 				<div id="unified-search"></div>
 				<div id="notifications"></div>
-				<div id="contactsmenu"></div>
 				<div id="user-menu"></div>
 			</div>
 		</header>
